python/fledge/common/service_record.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, s_id, s_name, s_type, s_protocol, s_address, s_port, m_port):
             self._port = int(s_port)
         self._management_port = int(m_port)
         self._status = ServiceRecord.Status.Running
+        self._debug = {}
```

Comment:
@MarkRiddoch As debug output for both Southbound and Northbound microservices returns `{"debug": {"debugger": "Detached"}}`. Should we keep this as the default, or is it fine to leave it empty? 
Please confirm.

Reply:
I suspect the GIU will find it easier if the they only have to look for one value rather than empty or a specific value.

Reply:
Currently, the API returns a debug key-value pair when the object is non-empty, so it's not an issue for them to find it on their own.